config/dictionaries/device.json

```diff
@@ -1,389 +1,1361 @@
+      },
+      {
+        "i": "Remove the balenaOS installation media."
+      },
+      {
+        "i": "Remove and re-connect power to the board."
+      }
+    ]
   },
   {
     "id": "generic-aarch64",
     "name": "Generic AARCH64 (ARMv8)",
     "arch": "aarch64",
+    "bootMedia": "SD card",
+    "state": "EXPERIMENTAL",
+    "machine": "generic-aarch64",
+    "community": false,
     "icon": "/img/device/generic-aarch64.svg",
-    "type": "virtual"
-  },
-  {
-    "id": "generic-armv7ahf",
-    "name": "Generic ARMv7-a HF",
-    "arch": "armv7hf",
-    "icon": "/img/device/generic-armv7ahf.svg",
-    "type": "virtual"
+    "instructions": [{
+        "i": "Download/pull the following docker image on the target machine: <code>{{ dockerImage }}</code> ."
+      },
+      {
+        "i": "Download the configuration file and deploy it on target."
+      },
+      {
+        "i": "Run the resinOS container using <a href=\"https://github.com/resin-os/resinos-in-container\">resinos-in-container</a>. This tool/script will reference the image and the configuration file (downloaded above) as command line arguments. Example: <code>./resinos-in-container.sh --config ./config.json --image {{ dockerImage }}</code>. Tweak the <code>--config</code> argument based on the filename set for the downloaded configuration file."
+      }
+    ]
   },
   {
     "id": "coral-dev",
     "name": "Google Coral Dev Board",
     "arch": "aarch64",
+    "bootMedia": "eMMC",
+    "state": "EXPERIMENTAL",
+    "machine": "coral-dev",
+    "community": false,
     "icon": "/img/device/coral-dev.svg",
-    "link": "https://coral.ai/products/dev-board/",
-    "type": "beta"
-  },
-  {
-    "id": "hummingboard",
-    "name": "Hummingboard",
-    "arch": "armv7hf",
-    "bootMedia": "SD card",
-    "icon": "/img/device/hummingboard.svg",
-    "link": "https://www.solid-run.com/nxp-family/hummingboard/",
-    "type": "released"
+    "instructions": [{
+        "i": "Set the BOOT_SELECT switch to the SD-CARD position."
+      },
+      {
+        "i": "Write the OS file you downloaded to your SD card. We recommend using <a href=\"http://www.etcher.io/\">Etcher</a>."
+      },
+      {
+        "i": "Insert the freshly burnt SD card into the Google Coral Dev Board."
+      },
+      {
+        "i": "<strong role=\"alert\">Warning!</strong> This will also completely erase internal storage media, so please make a backup first."
+      },
+      {
+        "i": "The device is performing a shutdown. Please wait until all LEDs are off."
+      },
+      {
+        "i": "Remove the balenaOS installation media."
+      },
+      {
+        "i": "Set the BOOT_SELECT switch to the eMMC position."
+      },
+      {
+        "i": "Remove and re-connect power to the board."
+      }
+    ]
   },
   {
     "id": "intel-edison",
     "name": "Intel Edison",
     "arch": "i386",
-    "bootMedia": "eMMC",
+    "bootMedia": "SD card",
```

Comment:
Looks like this is falling back to SD card due to a lack of instructions. In this case it won't matter as has custom content but I'll resolve this specific board.